### PR TITLE
Upgrades to node 6

### PR DESCRIPTION
**Pull request by vwraposo**
It also removes catalog graphql client in favor of the one in node-vtex-api

Depends on: https://github.com/vtex/node-vtex-api/pull/329

### CODE REVIEW

File: node/typings.ts [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-  interface State extends BroadcasterEvent {
+  interface State extends RecorderState, BroadcasterEvent {
     alwaysNotify: boolean
   }
 
   interface BroadcasterEvent {
     HasStockKeepingUnitModified: boolean
     IdSku: string
   }
+  interface Custom extends ParamsContext {
```
Comment (NataliaGodot) [inline node/typings.ts]: Why create a `Custom` empty interface instead of using `ParamsContext` directly?

File: node/package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "@types/co-body": "^0.0.3",
     "@types/node": "12.x",
     "@types/ramda": "types/npm-ramda#dist",
-    "@vtex/api": "^3.62.4",
+    "@vtex/api": "6.13.0",
```
Comment (NataliaGodot) [inline node/package.json]: Shouldn't you keep the caret to get bug fixes?

Comment (vwraposo) [reply node/package.json]: It was automatic, now the version is the same from the runtime. Therefore I do not think we need the caret, but i will check with @tiagonapoli

File: node/typings.ts (hunk 1 above)
Comment (vwraposo) [reply node/typings.ts]: That is unnecessary, will change 👍

File: node/index.ts [hunk 3]
```diff
@@ -59,6 +40,9 @@ export default new Service<Clients, State>({
       segment: {
         memoryCache: segmentCacheStorage,
         timeout: TREE_SECONDS_MS
+      },
+      catalogGraphQL: {
+        memoryCache: catalogCacheStorage,
```
Comment (tlgimenes) [inline node/index.ts]: I think we don't need a cache since we always want the data fresh

**Pull request merged**